src/CommonGLPI.php

```diff
@@ -1080,7 +1080,7 @@ class='btn btn-sm btn-icon btn-ghost-secondary " . ($prev >= 0 ? '' : 'bs-invisi
                 ]);
                 if ($this->isField('is_deleted') && $this->fields['is_deleted']) {
                     $title = $this->isField('date_mod')
-                                ? sprintf(__s('Item has been deleted on %1$s'), Html::convDateTime($this->fields['date_mod']))
+                                ? sprintf(__s('Item has been deleted on %s'), Html::convDateTime($this->fields['date_mod']))
```

Comment:
Why this change? Original should work, changing requires translation validation.

Reply:
Ok, it's a duplicate :)

Reply:
To use the same as here: https://github.com/glpi-project/glpi/blob/8fd103f268542027b178e91ba7bf98c74e73f811/templates/components/form/header_content.html.twig#L111